frontend-react/src/components/Content/Templates/IAMeta.tsx

```diff
@@ -0,0 +1,39 @@
+import { Helmet } from "react-helmet";
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+
+import { ContentDirectory, getDirectoryElement } from "../MarkdownDirectory";
+
+export interface IARouterProps {
+    directories: ContentDirectory[];
+    indexElement?: JSX.Element;
+}
+export const IARouter = ({ directories, indexElement }: IARouterProps) => {
+    return (
+        <Routes>
```

Comment:
The old use of generated routing:
```typescript
<Routes>
    <GeneratedRoutes />
    // 404 catch route
</Routes?
```

This doesn't work anymore because the only component name allowed between `Routes` is `Route` so the `GeneratedRoutes` component, despite being made of all Route components, doesn't work.